addons/mail/static/src/models/partner/partner.js

```diff
@@ -119,6 +158,55 @@ function factory(dependencies) {
             callback(partners);
         }
 
+        /**
+         * Returns partners that match the given search term.
+         *
+         * @static
+         * @param {string} searchTerm
+         * @param {Object} [options={}]
+         * @param {mail.thread} [options.thread] prioritize and/or restrict
+         *  result in the context of given thread
+         * @returns {mail.partner[]}
```

Comment:
`@returns {[mail.partner[], mail.partner[]]}`

Reply:
Good catch, the only one of the 4 were we actually use the extra list I managed to forget the 2nd param on it.